src/protocol/commands.rs

```diff
@@ -223,6 +223,13 @@ commands! {
         "qXfer:memory-map:read" => _qXfer_memory_map::qXferMemoryMapRead,
     }
 
+    host_io use 'a{
+        "vFile:open" => _vFile_open::vFileOpen<'a>,
+        "vFile:close" => _vFile_close::vFileClose,
+        "vFile:pread" => _vFile_pread::vFilePread,
+        "vFile:setfs" => _vFile_setfs::vFileSetfs,
```

Comment:
instead of checking for `:` in the parser for each of these operations, I'd just include `:` as part of the initial prefix search.

```suggestion
        "vFile:open:" => _vFile_open::vFileOpen<'a>,
        "vFile:close:" => _vFile_close::vFileClose,
        "vFile:pread:" => _vFile_pread::vFilePread,
        "vFile:setfs:" => _vFile_setfs::vFileSetfs,
```

Reply:
But all the other commands don't include the `:`. You could search `[b':',` in the source code.

Reply:
this is a very fair point...

alright, let's leave it as-is for now, and I can reevaluate whether I want to push `:` into the prefix match in a separate commit.